docs/components.md

```diff
@@ -21,62 +21,83 @@ We are using [lerna.js](https://lernajs.io/) to manage our packages and publish
 
 Lerna is a tool that optimizes the workflow around managing multi-package repositories with git and npm.
 
+It manages dependencies between our components so that if the version number of `globals` is bumped, all components dependent on `globals` will also have their version bumped.
+
+To do: We need to do more work to clarify how versioning, including managing breaking changes, will work.
+
 ### Publishing components manually (while in Alpha)
 
 In a new branch:
 
-- Run the `npm run build:packages` task to ensure any changes to `src/components/component-name` are copied to `packages/component-name`.
+1. Run the `npm run build:packages` task.
+
+2. Ensure all changes to `src/components/component-name` have been copied to `packages/component-name`.
 
+3. For any new components, add a `package.json` inside `packages/component-name`:
 ```
-npm run build:packages
+{
+    "name": "@govuk-frontend/component-name",
+    "version": "version-number",
+    "dependencies": {
+      "@govuk-frontend/globals": "version-number",
+      "@govuk-frontend/icons": "version-number"
+    }
+}
 ```
+Note: Discuss with your team how you should handle versioning when new components
+are added.
 
-- Ensure each component has a package.json file listing its dependencies.
-
-- Commit changes to the updated packages
+4. Include new components in `packages/all/package.json`.
 
+5. Commit the changes to the updated packages
 ```
 git add packages/*
 git commit -m "chore(packages): copy changes to packages"
 ```
 
-Open a pull request for these changes.
-
+6. Open a pull request for these changes.
 
-Once these have been merged into master.
+Once the PR has been merged into master:
 
-- Run lerna bootstrap to install all package dependencies and link any cross-dependencies
+1. Log in to `npm` as authorised user.
```

Comment:
As a user who has permission to publish to the govuk-frontend scope?

Reply:
Yup, good shout.